### ISSUE TITLE
More HTML documentation?

### ISSUE THREAD

**Issue by SteveAtKlover** [flutter-view/flutter-view#44]
Hi, I was under the impression that you can write html and sass to create flutter views, but most of the documentation only covers pug. Do I need to use pug or can I just use html? I see if I save an html file, it will generate a corresponding dart file, but the generated dart file contains errors.

**Comment by blueneogeo** [comment]
Hi Steve,

You are correct that currently the HTML processing has bugs. In the documentation I corrected to use Pug, but the website still erroneously shows that you can use HTML as well, I will change that.

The HTML conversion had some issues that caused me to drop full support. Is it something that would benefit you over Pug? I found that the syntax of pug and css both using the same structure make it very easy to reason about, but use cases obviously differ, as do tools. If you greatly prefer HTML I may look into it.

**Comment by SteveAtKlover** [comment]
Yeah. I don't know pug syntax, so it's another barrier to using the app. I
will have to learn it if I intend to use flutter-view.
